lms/djangoapps/courseware/views/index.py

```diff
@@ -482,12 +484,21 @@ def _create_courseware_context(self, request):
                 table_of_contents['previous_of_active_section'],
                 table_of_contents['next_of_active_section'],
             )
-            courseware_context['unit'] = section_context.get('activate_block_id', '')
+            unit_id = section_context.get('activate_block_id', '')
+            courseware_context['unit'] = unit_id
             courseware_context['fragment'] = self.section.render(self.view, section_context)
 
             if self.section.position and self.section.has_children:
                 self._add_sequence_title_to_context(courseware_context)
 
+        # Courseware MFE link
+        if self.section and show_courseware_mfe_link(request.user, staff_access, self.course.id):
```

Comment:
As mentioned elsewhere, the MFE can decide on a section to show itself if need be... it may be we don't need to check for `self.section` here?  Right now, it'll just display the first section in the first chapter.  Ideally it'd be able to use the active/current block ID, but we don't have that in our API responses today.

Reply:
So I'll adjust the code because what you're saying makes sense, but I'm not sure if we can actually display this view in the existing LMS without a section being assigned (if one isn't supplied by the URL, it will try to load one from examining the modulestore).